invenio_rdm_records/services/schemas/metadata.py

```diff
@@ -140,12 +140,12 @@ class VocabularySchema(Schema):
 
     id = SanitizedUnicode(required=True)
     title = fields.Dict(dump_only=True)
+    props = fields.Dict(dump_only=True)
```

Comment:
Is the reason they are needed here, because otherwise you can create the facet over them?

Reply:
No, actually that was a leftover, since we expect these fields only to be indexed and we don't need to persist them in the DB I will remove them from the schema.